server/controllers/users.js

```diff
@@ -151,13 +151,46 @@ class UserController {
     }
   }
 
+  /**
+  * Get users and their corresponding profiles
+  * @async
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async getUsers(req, res, next) {
+    try {
+      const users = await User.findAll({
```

Comment:
this should include

```suggestion
      const users = await User.findAll({
        ...
        where: {
          active: true
        }
      ...
```

we don't need to return the `active` label

Reply:
Resolved changes